src/components/ServicesPage/Prices/index.tsx

```diff
@@ -0,0 +1,146 @@
+"use client";
+import React from "react";
+import SessionPriceCard from "@/components/ServicesPage/SessionPriceCard";
+import PhotographySection from "@/components/ServicesPage/PhotographySection";
+import { useState } from "react";
+
+const portraitPhotography = [
+  {
+    title: "Portrait Photography",
+    description:
+      "Our portrait photography service is all about showcasing your unique personality. Whether you need a professional headshot, a family portrait, or a personal photoshoot, we create images that reflect your true self. We work closely with you to bring out your best angles and expressions, ensuring every portrait tells your story.",
+    linkText: "View Projects",
+    linkUrl: "/projects",
+    imageUrl: "/kytsya.jpg",
+  },
+];
+
+const Prices: React.FC = () => {
+  const [currentSection, setCurrentSection] = useState(0);
+
+  const handlePrev = () => {
+    setCurrentSection((prev) =>
+      prev > 0 ? prev - 1 : portraitPhotography.length - 1
+    );
+  };
+
+  const handleNext = () => {
+    setCurrentSection((prev) =>
+      prev < portraitPhotography.length - 1 ? prev + 1 : 0
+    );
+  };
+
+  return (
+    <>
```

Comment:
you don't need fragment here

Reply:
JSX expressions must have one parent element.ts(2657)